### ISSUE TITLE
Deprecate the tool?

### ISSUE THREAD

**Issue by aniav** [halloffame/ynab-csv#30]
Hey!
I have updated this tool in my fork to make it work with the YNAB web version. It seems they are also linking to my version in the docs: https://docs.youneedabudget.com/article/921-importing-a-csv-file

People are filling issues in your version, which is not really maintained. Maybe it would be good to point out that the supported version is the one I am holding? I have also added support for a few things people have asked in the issues around here, like quoting the values and ability to choose the CSV encoding and I intend to maintain my fork.

Thanks for everything, it was a lot easier building on top of a tool already prepared 🙏 🌟

**Comment by benallfree** [comment]
Agreed. Contributed my PR to @aniav's fork https://github.com/aniav/ynab-csv/pull/26
